### PR TITLE
Test Driven Development

### PR DESCRIPTION
**Pull request by AshwinCarvalho999**
## In this project, we implemented the following:

- Created a class called Solver.
- Created a method called factorial that takes one argument, an integer N, and returns the factorial for that number
- Created a method called reverse that takes one argument, a string word, and returns the word, reversed
- Created a method called fizzbuzz that takes one argument, an integer N, and returns a string

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,206 @@
...
+> Describe between 1-3 key features of the application.
+
+- **List Books**
+- **List people**
+- **Create person & book**
+=======
+- On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)
+- On startup of our app it load all the data from files.
+>>>>>>> development
+
```
Comment (julie-ify) [inline README.md]: - [ ] Your project title, description, and key features are incorrect it seems. This is a TDD project but you are describing the School Library project

File: spec/solver_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    it 'When N is divisible by 3, return fizz' do
+      expect(solver.fizzbuzz(3)).to eql('fizz')
+    end
+
+    it 'When N is divisible by 5, return buzz' do
+      expect(solver.fizzbuzz(5)).to eql('buzz')
+    end
+
+    it 'When N is not divisible by 3 or 5' do
+      expect(solver.fizzbuzz(2)).to eql(2)
```
Comment (julie-ify) [inline spec/solver_spec.rb]: - [ ] When the number is indivisible by neither 3 nor 5 a string version of the number should be returned, not the number as it is. This should be `...to eql('2')`.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AshwinCarvalho999 and @j-tee,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @AshwinCarvalho999, and @j-tee👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**